src/shared/common/components/stage-summary.jsx

```diff
@@ -56,10 +56,6 @@ const StageSummary = () => {
         }
     }, [apiStatus]);
 
-    // load summary on mount
```

Comment:
again... have we taken this out on FOI for a reason or new in on `main`?

Reply:
It was taken out on epic/foi-mvp. It looks like this is because it happens anyway in form-enabled.jsx, so this would just trigger an unnecessary update. The other effect remains so that if there's an API update, it will be retriggered.

If during testing we have any issues with summaries being up to date, we can revisit this.